pkg/api/error.go

```diff
@@ -39,6 +42,11 @@ func handleFulcioAPIError(w http.ResponseWriter, req *http.Request, code int, er
 		message = http.StatusText(code)
 	}
 
-	log.RequestIDLogger(req).Errorw("exiting with error", append([]interface{}{"handler", req.URL.Path, "statusCode", code, "clientMessage", message, "error", err}, fields...)...)
+	// log.RequestIDLogger(req).Errorw("exiting with error", append([]interface{}{"handler", req.URL.Path, "statusCode", code, "clientMessage", message, "error", err}, fields...)...)
```

Comment:
delete?  I see the function below, but I'm not sure where you are calling it.

Reply:
yes, this will get deleted when I rip out the rest of the previous HTTP-centric code path